wp/plugin/wp.ml

```diff
@@ -228,6 +232,12 @@ module Cmdline = struct
             at each jump in the path. The path contains information about whether \
             a jump has been taken and the address of the jump if found."
 
+  let fun_input_regs = param bool "fun-input-regs" ~default:true
+      ~doc:"If set, at a function call site, uses all possible input registers \
+            as arguments to a function predicate generated for an output register \
```

Comment:
"function predicate [...] that holds the result of the function call" might be better called a "function symbol [...] that represents the result of the function call"? YMMV